config/peerpods/podvm/azure-podvm-image-handler.sh

```diff
@@ -555,6 +612,7 @@ function delete_image_gallery() {
     # Check if the gallery has any image versions
     get_all_image_ids
 
+    # This will set the IMAGE_ID_LIST variable
```

Comment:
`IMAGE_ID_LIST` is set by `get_all_image_ids`. Maybe move the comment before the call to the function.